include/albatross/Ransac

```diff
@@ -13,6 +13,10 @@
 #ifndef ALBATROSS_RANSAC_H
 #define ALBATROSS_RANSAC_H
 
-#include "models/ransac.h"
+#include "Core"
```

Comment:
Hmm... interesting.  Totally legit, but I always like to make things consistent; looks like we've done <albatross/Core> in other places, like Tune.

Reply:
which is better.  If it's a relative path should it be `"Core"`? or `<albatross/Core>`

Reply:
I think "Core", since it actually is local.  Unless you think these files might move into different directories.